requirements/pytorch/strategies.txt

```diff
@@ -2,4 +2,5 @@
 #  in case you want to preserve/enforce restrictions on the latest compatible version, add "strict" as an in-line comment
 
 deepspeed>=0.6.0, <0.8.0  # TODO: Include 0.8.x after https://github.com/microsoft/DeepSpeed/commit/b587c7e85470329ac25df7c7c2521ff9b2833db7 gets released
-lightning-colossalai==0.1.0dev1
+lightning-colossalai>=0.1.0dev1
+lightning-hivemind>=0.1.0dev0
```

Comment:
```suggestion
```
We don't need to install this one in our CI. There aren't any tests with it, right?